docs/reference/requirement-sets/word-api-requirement-sets.md

```diff
@@ -27,7 +27,9 @@ Word add-ins run across multiple versions of Office, including Office 2016 or la
 | [WordApi 1.1](word-api-1-1-requirement-set.md) | Version 1509 (Build 4266.1001) or later| January 2016, 1.18 or later | January 2016, 15.19 or later| September 2016 |
 
 > [!NOTE]
-> The build number for Office 2016 installed via MSI is 16.0.4266.1001. This version only contains the WordApi 1.1 requirement set.
+> Users on perpetual versions of Office will be restricted to certain requirement set.
+>
```

Comment:
Shouldn't we also include a bullet point for Office 2019? i.e., 

Office 2019 supports WordApi 1.3 and earlier.